app/views/two_factor_mailer/backup_code.html.erb

```diff
@@ -0,0 +1,20 @@
+<p>Hi <%= @user&.username %>,</p>
+
+<h3>Your two-factor authentication backup code</h3>
+
+<p>
+  You're receiving this email because you have two-factor authentication enabled on your Codidact network account. We've
+  recently added backup codes for every account using 2FA, so that you can still log in if you lose access to your
+  authenticator app.
+</p>
+
+<p>
+  Your backup code can be found in your user account settings, under Two-Factor Authentication settings. Please save it
+  somewhere safe. Please note that logging in using this backup code will disable 2FA on your account so that you can
+  reconfigure it using a new app or device &mdash; this will also reset your backup code.

```

Comment:
Can we add a link here such that it is easier for the user to get to it:

```suggestion
  Your backup code can be found in your user account settings, under
  <a href="<%= two_factor_status_path %>">Two-Factor Authentication settings</a>. Please save it somewhere safe.
  Please note that logging in using this backup code will disable 2FA on your account so that you can reconfigure it
  using a new app or device &mdash; this will also reset your backup code.
```